src/v/debug_bundle/debug_bundle_service.h

```diff
@@ -154,6 +155,8 @@ class service final : public ss::peering_sharded_service<service> {
 private:
     /// Handler used to emplace stdout/stderr into a buffer
     struct output_handler;
+    /// KV store used to store metadata
+    storage::kvstore* _kvstore;
```

Comment:
comment/question: The pattern I've usually seen is to pass a pointer or reference to the sharded service down into dependent modules. Is there a reason or advantage to doing it this way?